cmd/server/message.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+// Message represents a message in the chat server
+type Message struct {
```

Comment:
Message should be an interface, there might be a lot of implementations for messages (text, chat, voice etc)

Reply:
client should know how to show each message, think about how to implement it :)